src/test/java/spoon/test/method/MethodTest.java

```diff
@@ -85,4 +90,26 @@ public void testGetAllMethods() throws Exception {
 		assertEquals(1, methods.stream().filter(method -> "foo".equals(method.getSimpleName())).count());
 	}
 
+	@Test
+	public void testCloneMethodUsingjlObjectMethods() throws Exception {
```

Comment:
What does  the **jl** mean in this test name? testCloneMethodUsing**jl**ObjectMethods